api/metric/counter.go

```diff
@@ -30,41 +30,41 @@ type Int64Counter struct {
 	commonMetric
 }
 
-// Float64CounterHandle is a handle for Float64Counter.
+// Float64CounterBoundInstrument is a boundInstrument for Float64Counter.
```

Comment:
The spec uses the terms `bound instrument` in comments rather than `boundInstrument`, should I also follow that convention?

Reply:
I'll suggest BoundFloat64Counter instead of Float64CounterBoundInstrument

Reply:
Since Bound can be used as verb and a noun , 'Bound' in 'BoundFloat64Counter' seems more like a verb than a noun. Float64CounterBoundInstrument is long.  Maybe BoundedFloat64Counter?

Reply:
I would use "bound instrument" in sentences such as this, yes.

"Bound" can be a verb, but not one that makes sense in this context.  The first definition for Bound as an adjective that I found makes sense though, and I would suggest we use it ([Fastened by or as if by a band](https://www.merriam-webster.com/dictionary/bound)]). "Bounded" means something quite different in this context, to me (the connotation is "to have bounds applied", e.g., describing limits).

I think I prefer "BoundFloat64Counter" instead of "Float64CounterBoundInstrument", which is what the OTEP recommends.  See: https://github.com/open-telemetry/oteps/blob/master/text/0070-metric-bound-instrument.md#internal-details